src/ILCompiler.ReadyToRun/src/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -362,6 +362,9 @@ private ISymbolNode GetHelperFtnUncached(CorInfoHelpFunc ftnNum)
                 case CorInfoHelpFunc.CORINFO_HELP_UNBOX_NULLABLE:
                     id = ReadyToRunHelper.Unbox_Nullable;
                     break;
+                case CorInfoHelpFunc.CORINFO_HELP_GETREFANY:
```

Comment:
Nit: other uninteresting helpers that throw `RequiresRuntimeJitException` are grouped all the way down. Could save two lines.

Reply:
Good point, moved in 2nd commit.